src/cli/load.js

```diff
@@ -177,7 +191,7 @@ export type SaveGraph = (
   RepoId
 ) => Promise<Graph>;
 
-export type SaveCred = (Graph, RepoId) => Promise<void>;
+export type SaveCred = (Graph, RepoId, string | null) => Promise<void>;
```

Comment:
To make this a little clearer to the reader, how about we make it:

```js
export type SaveCred = (graph: Graph, repoId: RepoId, weightsPath?: string) => Promise<void>
```

this way someone reading the type signature can see what the string argument means. (Another option would be to make a type alias `export type WeightsPath = string)

Using `weightsPath?` means that it's an optional arg, so we won't need to do the `| null` stuff.

Reply:
Changed it.